pkg/util/testing/wrappers.go

```diff
@@ -62,6 +62,33 @@ func MakeJob(name, ns string) *JobWrapper {
 		},
 	}}
 }
+func MakeSleepJob(name, ns string) *JobWrapper {
```

Comment:
much simpler to just add a function `Image(image, args...)` to the JobWrapper :)